docker-compose.yml

```
@@ -1,18 +1,24 @@
 version: '3'
 services:
   redis:
+    container_name: redis
     image: redis
     ports:
       - "6379:6379"
 
   sidekiq:
+    container_name: sidekiq
     build: .
     command: bundle exec sidekiq
     depends_on:
       - redis
+    environment:
+      - REDIS_URL=redis://redis:6379/0
```

This syntax is nicer/better readable IMO:

```suggestion
      REDIS_URL: redis://redis:6379/0
```